tests/core/api/GH_ORG_SUBS_API.js

```diff
@@ -0,0 +1,436 @@
+              logger.error(test, 'Failed to setup tests. err:', err);
+              return done(err);
+            }
+            ownerApiAdapter =
+              global.newApiAdapterByStateAccount('ghOwnerAccount');
+            collaboraterApiAdapter =
+              global.newApiAdapterByStateAccount('ghCollaboratorAccount');
+            memberApiAdapter =
+              global.newApiAdapterByStateAccount('ghMemberAccount');
+            unauthorizedApiAdapter =
+              global.newApiAdapterByStateAccount('ghUnauthorizedAccount');
+
+            successStatusCode = _.findWhere(global.systemCodes,
+              {group: 'statusCodes', name: 'SUCCESS'}).code;
+
+            ownerApiAdapter.getProjects('',
+              function (err, prjs) {
+                if (err || _.isEmpty(prjs))
+                  return done(new Error('Project list is empty', err));
+
+                project = _.first(
+                  _.where(prjs, {isOrg: true, isPrivateRepository: true}
+                  )
+                );
+
+                project.test_resource_type = 'project';
+                project.test_resource_name = 'ghOrgPrivate';
+                assert.isNotEmpty(project, 'User cannot find the project');
+                return done();
+              }
+            );
+          }
+        );
+      }
+    );
+
+    it('1. Owner can get all its subscriptions',
+      function (done) {
+        ownerApiAdapter.getSubscriptions('',
+          function (err, subs) {
+            if (err || _.isEmpty(subs))
+              return done(
+                new Error(
+                  util.format('User cannot get subscription',
+                    query, err)
+                )
+              );
+            subscriptions = subs;
+            assert.isNotEmpty(subscriptions, 'User cannot find the subscriptions');
+            assert.equal(subscriptions.length,
+              global.ADM_GH_SUB_COUNT, 'Subscription count needs to match');
+            return done();
+          }
+        );
+      }
+    );
+
+    it('2. Owner can get subscription by ID',
+      function (done) {
+        var sub = _.findWhere(subscriptions, {orgName:global.TEST_GH_ORGNAME}) || {};
+        assert.isNotEmpty(sub,
+          'Subscriptions cannot be empty.');
+        ownerApiAdapter.getSubscriptionById(sub.id,
+          function (err, subAcct) {
+            if (err || _.isEmpty(subAcct))
+              return done(
+                new Error(
+                  util.format('User cannot get subscription Account from ID',
+                    query, err)
+                )
+              );
+            subscription = subAcct;
+            assert.isNotEmpty(subscription, 'User cannot find the subscription by ID');
+            return done();
+          }
+        );
+      }
+    );
+
+    it('3. Member can get subscription by ID',
+      function (done) {
+        var sub = _.findWhere(subscriptions, {orgName:global.TEST_GH_ORGNAME}) || {};
```

Comment:
same issue here, we should use the same project and make sure all the folks can get it.